docs/_docs/reference/contextual/givens.md

```diff
@@ -14,38 +14,35 @@ trait Ord[T]:
     def < (y: T) = compare(x, y) < 0
     def > (y: T) = compare(x, y) > 0
 
-given intOrd: Ord[Int] with
+given intOrd: Ord[Int]:
   def compare(x: Int, y: Int) =
     if x < y then -1 else if x > y then +1 else 0
 
-given listOrd[T](using ord: Ord[T]): Ord[List[T]] with
+given listOrd: [T: Ord] => Ord[List[T]]:
```

Comment:
Why not using a `as` clause here? Then there'd be no need for the `summon` below.

Reply:
It's because we introduce `as` clauses only on the next page.